wasp/apps/alarm.py

```diff
@@ -136,6 +145,16 @@ def background(self):
         del self.day_btns
 
         self._set_pending_alarms()
+        try:
+            if self.num_alarms == 0:
+                return
+            with open("alarms.txt", "w") as f:
+                for n in range(self.num_alarms):
+                    al = self.alarms[n]
+                    f.write(",".join(map(str, al)) + ";")
+        except Exception:
+            pass
```

Comment:
why are exceptions ignored here? shouldn't the user be aware of any write failures?

Reply:
I recently had very annoying IO failures because I didn't have space left because of my sleep logging. I ended up leaving it like that but am still all ears.

Reply:
I think in that case the user should be alerted. If I expect the alarm to be saved and someday it silently doesn't happen I could miss an alarm

Reply:
For example by simply opening a PagerApp with a message like "Warning: unable to save your alarm to flash storage, it will not persist a reboot."

Reply:
Thanks!
See  https://github.com/daniel-thompson/wasp-os/pull/350